docs/eventing/experimental-features.md

```diff
@@ -81,10 +94,13 @@ kind: InMemoryChannel
 name: my-channel
 ```
 
-With this feature you can allow Knative to resolve the full `APIVersion` and further upgrades, deprecations and removals of the referred CRD without affecting existing resources.
+With this feature you can allow Knative to resolve the full `APIVersion` and
+further upgrades, deprecations and removals of the referred CRD without
+affecting existing resources.
 
 !!! note
-    At the moment this feature is implemented only for `Subscription.Spec.Subscriber.Ref` and `Subscription.Spec.Channel`.
+    At the moment this feature is implemented only for
+`Subscription.Spec.Subscriber.Ref` and `Subscription.Spec.Channel`.
```

Comment:
```suggestion
    At the moment this feature is implemented only for `Subscription.Spec.Subscriber.Ref` and `Subscription.Spec.Channel`.
```
This note is not rendering properly in the preview; I think it's because of the unnecessary break across lines